apollo-demo/src/main/java/com/ctrip/framework/apollo/demo/spring/autoRefreshDemo/AutoRefreshDemo.java

```diff
@@ -0,0 +1,58 @@
+package com.ctrip.framework.apollo.demo.spring.autoRefreshDemo;
+
+import com.ctrip.framework.apollo.demo.spring.common.bean.AnnotatedBean;
+import com.ctrip.framework.apollo.demo.spring.common.bean.AutoRefreshBean;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.util.Scanner;
+
+/**
+ * @author tony jiang(258737400@qq.com)
+ */
+public class AutoRefreshDemo {
```

Comment:
This change impacts a lot, so we should have more tests to cover the value change scenarios.

Reply:
Thank you for your comment.

I'm going to improve the test code.